### ISSUE TITLE
Remove Unnecessary Data from the Base Dataset

### ISSUE THREAD

**Issue by yashwantherukulla** [csivitu/snake-spotter#1]
The downloaded base dataset contains directories that are not needed for the snake vs background classification task. Identify these unnecessary directories and remove them to get the proper dataset.

NOTE: This needs to be done before anything else. This issue has to be solved with code, manual deletion of directories will not be considered. 
Prerequisites: None

No need of any ML knowledge for this issue, try it out
